docs/freqai.md

```diff
@@ -794,3 +794,68 @@ Code review, software architecture brainstorming:
 Beta testing and bug reporting:
 @bloodhunter4rc, Salah Lamkadem @ikonx, @ken11o2, @longyu, @paranoidandy, @smidelis, @smarm,
 Juha Nykänen @suikula, Wagner Costa @wagnercosta
+
+## Using the `spice_rack`
+
+<!-- Dont forget this section during the doc reorg! -->
+
+The `spice_rack` is aimed at users who do not wish to deal with setting up `FreqAI` confgs, but instead prefer to interact with `FreqAI` similar to a `talib` indicator. In this case, the user can instead simply add two keys to their config:
```

Comment:
(mostly / in part copied from https://github.com/freqtrade/freqtrade/pull/7442#issuecomment-1253213755)

We imagine / advertise spice_rack to be "just another indicator you can use" - so there's nothing really preventing the following (which i think is a valid usecase, btw).

``` python
        dataframe['dissimilarity_index'] = self.freqai.spice_rack('DI_values', dataframe, metadata, self)
        informative_1h['dissimilarity_index'] = self.freqai.spice_rack('DI_values', informative_1h, metadata, self)
```

user would like to have the dissimilarity index - but calculated against the informative 1h pair - which may (or may not) have a different length.

---

we seem to place `spice_rack` as an (intelligent) indicator of sorts - just like ta-lib.
ta-lib can do this ... can spice_rack too (or should it)?

if not - i think we need to change what we're trying to bring here - as it's got quite some limitations ATM (only for the main dataframe, ...) which is not a problem i think.

we just need to be aware that users will find ways to abuse this - (which is a good thing, as this usually results in better code).

Reply:
Sounds like you have some ideas for how you would like this to look. Personally my time ends here on this PR. You are welcome to reopen it if you want it integrated.